example/lib/main.dart

```diff
@@ -30,14 +31,15 @@ class _PreloadPageViewState extends State<PreloadPageViewDemo> {
         appBar: AppBar(
           title: const Text("PreloadPageView Demo"),
         ),
-        body: Center(
+        body: Container(
             child: PreloadPageView.builder(
-          itemCount: 3,
-          itemBuilder: (BuildContext context, int position) {
-            new Text("item" + position.toString());
-          },
+          preloadPagesCount: 5,
+          itemBuilder: (BuildContext context, int position) =>
+              DemoPage(position),
```

Comment:
Looks good. Have you run this example in you phone?

Reply:
Yes, it did work on my phone. Tests would be nice though, we can include those from Flutter team and add some for preload checking.

Reply:
👍